### PR TITLE
timeslot sorting accounts for days

### PR DESCRIPTION
**Pull request by kenneth-miura**
There was a bug with ConflictFilter caused by the fact that the timeslot `compareTo` method only sorted by it's starting `LocalTime`. Added a change so it checks the `DayOfWeek` first, and then checks the `LocalTime` only if it is the same day.

### CODE REVIEW

**Comment by roryhofland** [comment]
not sure what it wasn't working for before, but I tested it a couple of times and it works.

File: src/main/java/entities/Timeslot.java [hunk 1]
```diff
@@ -100,6 +100,10 @@ public boolean equals(Object o) {
 
     @Override
     public int compareTo(Timeslot that) {
-        return this.start.compareTo(that.getStart());
+        int dayCompare = this.day.compareTo(that.getDay());
+        if (dayCompare == 0) {
+            return this.start.compareTo(that.getStart());
```
Comment (EclipseIsDead) [inline src/main/java/entities/Timeslot.java:105]: this seems unnecessary, especially when DateTime implements compare already

Comment (sugolov) [reply src/main/java/entities/Timeslot.java:105]: fine for now, should modify for phase 2.

**Review by sugolov — APPROVED** [review]
looks good.

**Pull request merged**